contrib/php-fpm.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/*
+## Installing
+
+Add to your _deploy.php_
+
+```php
+require 'contrib/php-fpm.php';
+```
+
+## Configuration
+
+- `php_fpm_version` – The PHP-fpm version. For example: `8.0`.
+- `php_fpm_service` – The full name of the PHP-fpm service. Defaults to `php{{php_fpm_version}}-fpm`.
+- `php_fpm_command` – The command to run to reload PHP-fpm. Defaults to `echo "" | sudo -S /usr/sbin/service {{php_fpm_service}} reload`.
+
+## Usage
+
+Start by explicitely providing the current version of PHP-version using the `php_fpm_version`.
+Alternatively, you may use any of the options above to configure how PHP-fpm should reload.
+
+Then, add the `php-fpm:reload` task at the end of your deployments by using the `after` method like so.
+
+```php
+set('php_fpm_version', '8.0');
+after('deploy', 'php-fpm:reload');
+```
+
+ */
+namespace Deployer;
+
+set('php_fpm_version', function () {
+    throw new \Exception('Please provide the PHP-fpm version using the `php_fpm_version` option.');
```

Comment:
What adding detection of php-fpm version installed? And if two or more installed only then throw an exception?

Reply:
Good idea! I've managed to do it by looking at the running processes so even if multiple PHP-FPM versions exist, we default to using the one actually being used. 🙂

Reply:
Thank you for the invitation to the Deployer organization btw! 😊